sdg/OutputDocumentationService.py

```diff
@@ -0,0 +1,162 @@
+import os
+import sdg
+import numpy
+from slugify import slugify
+
+class OutputDocumentationService:
+    """HTML generation to document outputs built with this library.
+
+    Note that this is meant to document particular builds, not the library in
+    general. The idea is that each time this library is used to build/convert
+    some SDG-related data, this class can be used to generate human-friendly
+    HTML pages documenting the specifics of the build (such as endpoint URLs).
+    """
+
+
+    def __init__(self, outputs, folder='_site', branding='Build docs', languages=None, intro=''):
+        """Constructor for the OutputDocumentationService class.
+
+        Parameters
+        ----------
+        outputs : list
+            List of objects inheriting from OutputBase
+        """
+        self.outputs = outputs
+        self.folder = folder
+        self.branding = branding
+        self.intro = intro
+        self.slugs = []
+        self.languages = [] if languages is None else languages
+
```

Comment:
The order your functions are being called:

* `generate_documentation`
* `create_filename`
* `write_documentation`
* `write_index`
* `get_html`
* `write_page`

It'll probably make your code more readable if you define your functions in this order.